### DISCUSSION TITLE
Create & publish seperate package with changeset-related danger logic

### DISCUSSION THREAD

**Discussion by golship-liker** [twilio-labs/paste#2366] [Ideas] [answered]
Howdy, I was looking for a plugin that would allow to run logic with danger.js based on the absence/existence of changesets of publicly published packages. Closest thing to that is the code that exists in this repo's `.danger` folder. 

Is it possible for someone to create a seperate npm package with that functionality? I could also make it myself and I think the license in this repo allows for it, but still wanted to give a heads up. 🗿

**Comment by gloriliale** [comment, marked as answer]
Hi @coolerAaron! We're not planning on releasing it as a separate package at the moment, but please feel free to make your own. Thanks for the heads up 🗣️
